windows/ReactTestApp/MainPage.xaml

```diff
@@ -8,21 +8,42 @@
     mc:Ignorable="d"
     xmlns:react="using:Microsoft.ReactNative">
 
+    <Page.Resources>
+        <ResourceDictionary>
+            <SolidColorBrush x:Key="ButtonBackground" Color="{ThemeResource SystemAltHighColor}"/>
+        </ResourceDictionary>
+    </Page.Resources>
+
     <Grid>
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
 
-        <MenuBar VerticalAlignment="Top" Grid.Row="0">
-            <MenuBarItem Title="React" x:Name="MenuFlyout">
-                <MenuFlyoutItem Text="Load from JS bundle" Click="LoadFromJSBundle"/>
-                <MenuFlyoutItem Text="Load from dev server" Click="LoadFromDevServer"/>
-                <MenuFlyoutSeparator/>
-            </MenuBarItem>
-        </MenuBar>
+        <Grid x:Name="TitleBar">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
```

Comment:
Will this still work if we drop `ColumnDefinitions`? According to the docs, this will improve layout performance: https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/optimize-your-xaml-layout#use-single-cell-grids-for-overlapping-ui

Reply:
It will work now since we keep only one column.